tests/ludwig/config_validation/test_validate_config_misc.py

```diff
@@ -277,6 +255,13 @@ def test_defaults_schema():
     assert LOSS in schema.category.to_dict()
 
 
+def test_gbm_defaults_schema():
+    schema = GBMDefaultsConfig()
+    assert AUDIO not in schema.to_dict()
+    assert schema.category.encoder.dropout == 0.0
```

Comment:
@jeffkinnison @arnavgarg1 It just occurred to me - do we actually support encoders/decoders for GBM?